DeceptionGame/Assets/Scripts/EventManager.cs

```diff
@@ -3,64 +3,200 @@
 using UnityEngine;
 
 public class EventManager : InteractableObject {
```

Comment:
Not sure how I feel about the EventManager being an "Interactable Object" :/ I think the interactable logic and the state should be separated

Reply:
big agree